src/commands/Admin/settings.ts

```diff
@@ -0,0 +1,76 @@
+import {Args, PieceContext} from '@sapphire/framework';
+import {Message} from 'discord.js';
+import {BediEmbed} from '../../lib/BediEmbed';
+import {getSettings} from '../../database/settingsDB';
+import {capFirstLetterEveryWord} from '../../utils/stringsUtil';
+import {listModulesString, validModule} from '../../utils/settingsUtil';
+
+const {Command} = require('@sapphire/framework');
+
+module.exports = class SettingsCommand extends Command {
+  constructor(context: PieceContext) {
+    super(context, {
+      name: 'settings',
+      aliases: ['setting'],
+      description: 'Displays the current guild settings',
+    });
+  }
+
+  async run(message: Message, args: Args) {
+    const {guild, guildId} = message;
+
+    if (!guild) {
+      const embed = new BediEmbed()
+          .setColor('RED')
+          .setTitle('Settings Reply')
+          .setDescription('This command is only for guilds!');
+
+      return message.reply({
+        embeds: [embed],
+      });
+    }
+
+    let settingsData = await getSettings(guildId as string);
+
+    const module = await args.restResult('string');
+
+    const embed = new BediEmbed()
+        .setColor('BLUE')
+        .setTitle('Settings Reply');
+
+    if (!module.success || !validModule(module.value)) {
```

Comment:
I'm not a fan of the validModule implementation here.... the arguments/value pairs for what we offer in settings shouldn't have to be defined twice in my opinion